new-dti-website/src/hooks/useTitle.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react';
+
+const useTitle = (title?: string) => {
+  useEffect(() => {
+    document.title = `${title ? `${title} | ` : ''} Cornell DTI`;
+  }, []);
```

Comment:
```suggestion
  }, [title]);
```
I would include the title within the dependency array so it's not only on mount! If we every decide to change the title in the future and for styling!